docs/docs/concepts/features/selfservice.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Self-Service
+---
+
+ZITADEL allows users to perform certain tasks themselves.
+For these tasks we either provide an user interface, or the tasks can be initiated or completed through our APIs.
+
+It is important to understand that, depending on your use case, there will exist different user-types that want to perform different actions:  
+
+- `User` are the end-users of your application. Like with any CIAM solution, users should be able to perform tasks like register/join, update their profile, manage authenticators etc.There are certain actions that can be executed pre-login, yet others require the user to have a valid session.
+- `Manager` are users with a [special manager role within ZITADEL](/docs/concepts/structure/managers) and can perform administrative actions such as system configuration or granting access rights to users.
+
+All self-service interfaces are available in different [languages](/docs/guides/manage/customize/texts#internationalization).
+
+:::info
+ZTIADEL covers the typical "CIAM" self-service capabilities as well as delegated access management for multi-tenancy scenarios. Please refer to the section [Managers](#managers).
+:::
+
+## Registration
+
+:::info
+You can pre-select a given organization by passing the scope `urn:zitadel:iam:org:domain:primary:{domainname}` in the authorization request.
+This will force users to register only with the specified organization.
+Also the branding and login settings (e.g. Social Login Providers) are directly shown to the user.
+:::
+
+### Local account
+
+Allows unauthenticated and users, who are not yet in the given organization, to create an account (register) themselves.
+
+- Mandatory profile fields
+- Set password
+- Accept terms of service and privacy policy
+- User receives an email with an one-time code
+- User has to enter the one-time code to finish registration
+- User can re-quest a new one-time code
+
+The user is prompted on first login to setup MFA.
+This step is mandatory in case MFA is enforced in the login policy.
+
+### Existing Identity / SSO / Social Login
+
+Allows unauthenticated and users, who are not yet in the given organization, to register with an external identity provider.
+An external identity provider can be a Social Login Provider or a pre-configured identity provider.
+
+- Information from the external identity provider is used to pre-fill the profile information
+- User can update the profile information
+- An account is created within ZITADEL and linked with the external identity provider
+
+#### Account Linking
```

Comment:
I think we should mention autoregister / autolinking as well here

Reply:
Ok. Done.